heron/executor/src/python/heron_executor.py

```diff
@@ -102,6 +102,7 @@
 @click.option("--topology-defn-file", required=True)
 @click.option("--topology-id", required=True)
 @click.option("--topology-name", required=True)
+@click.option("--enable-verbose-gc-log", required=False)
```

Comment:
so you want to add a [boolean option to the click app](https://click.palletsprojects.com/en/7.x/options/#boolean-flags), which I think would be best as:
```suggestion
@click.option("--verbose-gc-log", is_flag=True)
```
Without the "enable" as that's implied, and using `is_flag` so that the parser stores a bool.